code/modules/mob/living/carbon/xenomorph/Castes/Predalien.dm

```diff
@@ -1,4 +1,4 @@
-/mob/living/carbon/Xenomorph/Predalien
+/mob/living/carbon/Xenomorph/Predalien/
```

Comment:
Why this change?

Reply:
A leftover from copypasting the path for procs.